src/components/deck-gl-wrapper/deck-gl-wrapper.tsx

```diff
@@ -761,21 +767,36 @@ export const DeckGlWrapper = ({
         currentViewport = viewport;
       }}
       {!showTerrain && (
-        <StaticMap
-          reuseMaps
+        <MaplibreMap
           mapStyle={mapStyle}
-          preventStyleDiffing
-          preserveDrawingBuffer
-        />
+          terrain={
+            showTerrain
+              ? { source: "dem-data-source", exaggeration: 1 }
+              : undefined
+          }
+        >
+          <MaplibreSource
```

Comment:
We don't need maplibre terrain now

Reply:
removed condition and kept terrain={undefined}